src/Sulu/Bundle/TagBundle/Tests/Functional/Controller/TagControllerTest.php

```diff
@@ -310,13 +310,9 @@ public function testDeleteById(): void
 
         $tag1Id = $tag1->getId();
 
-        $mockedEventListener = $this->getMockBuilder('Mock')->setMethods(['onDelete'])->getMock();
-        $mockedEventListener->expects($this->once())->method('onDelete');
-
-        $this->client->getContainer()->get('event_dispatcher')->addListener(
-            'sulu.tag.delete',
-            [$mockedEventListener, 'onDelete']
-        );
```

Comment:
The way to check if this method is called can be simplified by just passing a closure and setting a variable inside to true. That way we don't need to create a mock object.